Directory.Build.targets

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <ItemGroup>
+    <!-- ML.NET -->
+    <PackageReference Update="Microsoft.ML" Version="2.0.0-preview.22410.1" />
+    <PackageReference Update="Microsoft.ML.OnnxConverter" Version="0.20.0-preview.22410.1" />
```

Comment:
The builds I'm using here, are from:

https://dev.azure.com/dnceng/public/_artifacts/feed/MachineLearning